### DISCUSSION TITLE
nanoid customAlphabet

### DISCUSSION THREAD

**Discussion by Bigua** [usebruno/bruno#1167] [Q&A]
Hi!

in nanoid doc you can set a custom alphabet like this:
`import { customAlphabet } from 'nanoid';
const nanoid = customAlphabet('1234567890abcdef', 10);
model.id = nanoid() //=> "4f90d13a42"`

How can I use a custom alphabet in bruno script?

**Comment by GabrielRF** [comment]
Up
